Framework/PythonInterface/test/python/plugins/algorithms/CalculateSampleTransmissionTest.py

```diff
@@ -13,16 +13,16 @@ def test_sample_transmission_calculation(self):
 
         # Using water sample
         formula = "H2-O"
-        density = 0.033424
+        density = 0.100272
```

Comment:
As an additional point when Miguel was testing this these were the numbers he used. I assume this is be cause `0.1 mm` of water at `1 g/cc` is a number most instrument scientists will know.